dkg/pedersen/mod.go

```diff
@@ -378,14 +587,40 @@ func (a *Actor) Reshare(co crypto.CollectiveAuthority, thresholdNew int) error {
 	return nil
 }
 
-// Gets the list of the old committee members and new committee members and
-// returns the new committee members that are not common
-func substract(addrsSlice1 []mino.Address, addrsSlice2 []mino.Address) []mino.Address {
+// checkDecryptionProof verifies the decryption proof.
+//
+// See https://arxiv.org/pdf/2205.08529.pdf / section 5.4 Protocol / step 3
+func checkDecryptionProof(sp types.ShareAndProof, K kyber.Point) error {
+
+	tmp1 := suite.Point().Mul(sp.Fi, K)
```

Comment:
When we move this stuff to Kyber, it could benefit from a few comments, so that we don't have to read the paper every time to understand what the code is doing